### PR TITLE
fix: style issues on web

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
### Test

- `yarn`
- Test web and native

### CODE REVIEW

File: packages/app/components/card/card.tsx [hunk 1]
```diff
@@ -89,12 +92,15 @@ function Card({
...
+        style={[
+          {
+            // @ts-ignore
+            boxShadow: isDark ? CARD_DARK_SHADOW : CARD_LIGHT_SHADOW,
+          },
+          sizeStyle,
+        ]}
         tw={[
-          size,
+          !sizeStyle ? size : "",
```
Comment (intergalacticspacehighway) [inline packages/app/components/card/card.tsx:103]: not very happy about this solution. I think later we should convert media/card to explicitly accept height/width from outside.

**Comment by intergalacticspacehighway** [comment]
@alantoa the shadow issue is still there. Default shadow-lg of tailwind seems to be too much.

**Comment by alantoa** [comment]
@intergalacticspacehighway got it, thank you!
I will use tailwind config to replace the old shadow later.

**Review by alantoa — APPROVED** [review]
LGTM！💗

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

File: apps/next/tailwind.config.js [hunk 2]
```diff
@@ -319,6 +320,7 @@ module.exports = {
       },
       borderRadius: {
         inherit: "inherit",
+        4: "16px",
```
Comment (intergalacticspacehighway) [inline apps/next/tailwind.config.js:323]: @alantoa fixed the input rounded here.

**Pull request merged**